### PR TITLE
[forge] fix consensus stress test check

### PR DESCRIPTION
**Pull request by igor-aptos**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3902)

### CODE REVIEW

**Review by bchocho — COMMENTED** [review]
I'm a bit confused what we're doing here. So we expect all validators to have 1000 more txns after they have synced to target_v?

**Comment by igor-aptos** [comment]
no, we are checking that last 1000 transactions (before target_v, so that we know all nodes have them in history) are identical.

i.e. that nodes haven't diverged

File: testsuite/forge/src/test_utils/consensus_utils.rs [hunk 1]
```diff
@@ -137,29 +137,46 @@ pub async fn test_consensus_fault_tolerance(
     let transactions: Vec<_> =
         join_all(validator_clients.iter().cloned().map(move |v| async move {
             let mut txns =
-                v.1.get_transactions(None, Some(1000))
-                    .await
-                    .map_err(|e| anyhow::anyhow!("{:?}", e))?
-                    .into_inner();
+                v.1.get_transactions(
+                    Some(if target_v < 1000 { 0 } else { target_v - 1000 }),
```
Comment (sitalkedia) [inline testsuite/forge/src/test_utils/consensus_utils.rs:141]: nit: `cmp::max(target_v - 1000, 0)` would be concise and make it more obvious.

**Review by sitalkedia — APPROVED** [review]

**Review by bchocho — APPROVED** [review]
Yes my mind is working backwards

File: testsuite/forge/src/test_utils/consensus_utils.rs (hunk 1 above)
Comment (igor-aptos) [reply testsuite/forge/src/test_utils/consensus_utils.rs:141]: yeah, but those are u64, so I would need to cast.

but actually I can just do target_v.saturating_sub(1000) which is even clearer.

**Pull request merged**